rotary/config.yaml

```diff
@@ -15,6 +15,43 @@ minlength: 1000
 # Minimum average quality (PHRED) score. Q13 is ~5% expected error rate.
 minavgquality: 13
 
+## Short read QC
+## Adapter trimming
+# Perform adapter trimming from the right side of reads? Identifies adapters using both a built-in database.
+perform_adapter_trimming: "True"
+# K-mer length for adapter trimming (this k-mer length should be shorter than the adapters)
+adapter_trimming_kmer_length: 27
+# Minimum adapter length to be detectable on the (right-hand) end of a read. Lower number = more false positives
+minimum_detectable_adapter_length_on_read_end: 6
+# In addition to database-based trimming, trim the overhang of reads if they completely overlap
+#  (Requires at least a 40 bp insert and 14 bp read overlap)
+overlap_based_trimming: "True"
+# Minimum read length after adapter trimming (otherwise is discarded)
+minimum_read_length_adapter_trim: 10
+## Quality trimming
+# Whether to perform quality trimming on the right ("r") side of each read or on both the left and right ("rl") sides
+#  Otherwise, specify "f" to skip quality trimming altogether.
+quality_trim_direction: "rl"
+# Quality score cutoff for quality trimming
+quality_trim_score_cutoff: 20
+# Minimum average quality score required for a read after quality trimming (disabled if quality_trim_direction is "f")
+minimum_average_quality_score_post_trim: 20
+# Minimum read length after quality trimming (otherwise is discarded)
+minimum_read_length_quality_trim: 10
+## Contaminant filtration
+# Search for and remove reads that match contaminant reference sequences?
+perform_contaminant_filtration: "True"
```

Comment:
@jmtsuji What are your thoughts about removing this parameter and having it so that the sets both `contamination_references_ncbi_accessions` and `contamination_references_custom_filepaths` to `[]` to turn off the contamination filtering. Having this `perform_contaminant_filtration` flag is redundant now with your recent changes. What are your thoughts?

Reply:
Implement in https://github.com/jmtsuji/rotary/pull/82